Assets/Scripts/CameraShader.cs

```diff
@@ -1,32 +1,40 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.Rendering.PostProcessing;
 
 public class CameraShader : MonoBehaviour {
 
-  [SerializeField] private Shader shader;
-  [SerializeField] private Camera cam;
-  private bool shaderSet = false;
+    public PostProcessVolume volume;
```

Comment:
Could this be private instead?

Reply:
I think it is better for it to be public in case we decide to swap out for another post process effect later, or we change the post process effect name